components/vim/vim.p5m

```diff
@@ -1636,6 +1636,7 @@ link path=usr/share/man/ru.KOI8-R/man1/rview.1 target=vim.1
 link path=usr/share/man/ru.KOI8-R/man1/rvim.1 target=vim.1
 link path=usr/share/man/ru.UTF-8/man1/rview.1 target=vim.1
 link path=usr/share/man/ru.UTF-8/man1/rvim.1 target=vim.1
+file path=usr/share/vim/vimrc
```

Comment:
This could be delivered without changes to `Makefile` `install:` recipe like so:

````
file files/vimrc path=usr/share/vim/vimrc
````